dali/operators/reader/coco_reader_op.cc

```diff
@@ -44,136 +101,165 @@ Tensor (``m * category_id``).)code")
 
 If set to False, the bboxes are returned as [x, y, width, height].)code",
       false)
-  .AddOptionalArg("masks",
-      R"code(If set to True, segmentation masks are read and returned as polygons,
-represented by a list of coordinates.
-
-Each mask can be one or more polygons, and for a given sample, the polygons are represented by the
-following tensors:
+  .DeprecateArg("masks", false,  // deprecated since 0.28dev
+    "``masks`` argument is now deprecated. Please use ``polygon_masks`` instead "
```

Comment:
Maybe we should keep an info how the deprecated format looks like?

Reply:
done